front/lib/api/poke/plugins/data_sources/operations.ts

```diff
@@ -20,15 +26,15 @@ const doOperation = (op: OperationType, connectorId: string) => {
   );
 
   switch (op) {
-    case "STOP":
+    case "STOP: stop workflows with intent to resume shortly after (no DB changes)":
```

Comment:
Super ugly to have that be the tool ID, but path of least resistance give the current design...